package.json

```diff
@@ -93,24 +92,23 @@
     "@babel/preset-env": "^7.12.7",
     "@babel/preset-react": "^7.16.7",
     "@babel/preset-typescript": "^7.16.0",
-    "@testing-library/react-hooks": "^3.7.0",
```

Comment:
Hooks testing has been absorbed into `@testing-library/react`, so removing this dep.